### ISSUE TITLE
jekyll-auth requires a githib organisation account rather than a personal account

### ISSUE THREAD

**Issue by hossg** [benbalter/jekyll-auth#58]
It seems that jekyll-auth requires an organisation and/or team id, but both - I think - presuppose an organisation-type github account, rather than a personal account which seems an unnecessary restriction.

**Comment by parkr** [comment]
@hossg You grant access to a team within your organization. Are you asking about the use case of only granting yourself access?

**Comment by hossg** [comment]
Actually it's not just myself, since I have shared access to my repo with others, but it's using GitHub with a "personal plan" rather than an "organisation" plan (https://github.com/pricing) so the team and org ids required to config Jekyll auth don't exist.



—

**Comment by hossg** [comment]
OK - it turns out there's a trivial solution, and to be honest I think the "problem" was as much my understanding of github organisations as anything else.  I simply created a "dummy" organisation, against which jekyll-auth can authorise access.  (My misunderstanding was that I might need to convert my existing personal account to an organisation, which I believe is an option I saw somewhere on github; but you don't HAVE to do that, and can simply create a new organisation and add your existing account to that organisation).
